src/index.ts

```diff
@@ -56,28 +62,40 @@ export interface GetCallOptions extends CallOptions {
   autoCreate?: boolean;
 }
 
-export interface PushConfig {
-  pushEndpoint: string;
-  attibutes?: Map<string, string>;
+export interface Inventory {
+  callbacks?: Array<RequestCallback<string>>;
+  queued?: Array<{}>;
+  bytes: number;
+  nack?: Array<[string, number]>;
```

Comment:
I think this found its way here by mistake, there isn't a `nack` array on the publisher inventory.

Reply:
removed.